order-karma.html

```diff
@@ -0,0 +1,204 @@
+                        <option>Norwich</option>
+                        <option>Nottingham</option>
+                        <option>Oxford</option>
+                        <option>Perth</option>
+                        <option>Peterborough</option>
+                        <option>Plymouth</option>
+                        <option>Portsmouth</option>
+                        <option>Preston</option>
+                        <option>Ripon</option>
+                        <option>St Albans</option>
+                        <option>St Asaph</option>
+                        <option>St Davids</option>
+                        <option>Salford</option>
+                        <option>Salisbury</option>
+                        <option>Sheffield</option>
+                        <option>Southampton</option>
+                        <option>Stirling</option>
+                        <option>Stoke-on-Trent</option>
+                        <option>Sunderland</option>
+                        <option>Swansea</option>
+                        <option>Truro</option>
+                        <option>Wakefield</option>
+                        <option>Wells</option>
+                        <option>Westminster</option>
+                        <option>Winchester</option>
+                        <option>Wolverhampton</option>
+                        <option>Worcester</option>
+                        <option>York</option>
+                    </select>
+                </div>
+                <div class="field-container thirty-percent">
+                    <label for="postcode">Postcode <span class="requiredstar">*</span></label>
+                    <input type="text" name="postcode" id="postcode" required/>
+                </div>
+            </div>
+            <fieldset>
+                <legend>Select a colour</legend>
+                <label for="orange" class="radio">
+                    <input type="radio" name="color" value="orange" id="orange" required checked />
+                    <span class="radio-button"></span>
+                    Karma orange
+                </label>                
+                <label for="grey" class="radio">
+                    <input type="radio" name="color" value="grey" id="grey"/>
+                    <span class="radio-button"></span>
+                    Space Grey
+                </label>
+            </fieldset>
+            <label for="terms" class="terms">
+                <input type="checkbox" name="terms" value="true" id="terms"/>
+                <span class="checkbox"></span>
+                By placing your order you agree to Karma's <a href="#">Terms and Conditions</a>. <span class="requiredstar">*</span>
+            </label>
+            </div>
+            <button type="submit">Place my order</button>
+        </form>
+        </section>
+        <section class="image-container">
+            <img src="./img/store-image.jpg" alt="Girl using Karma in a cafe"/>
```

Comment:
It is not advisable to directly embed large images in your HTML code, as they slow down page load. It would be better to instead treat this image as a background of the section element and then set the image from CSS with the background-size: cover property applied.

See more info here: https://developer.mozilla.org/en-US/docs/Web/CSS/background-size

Reply:
We were advised to use flexbox to put the form and the image next to each other. I put the image as a background for mobile devices

Reply:
Great 👍